Automattic/wc-calypso-bridge
# Update store setup form styling

Fixes #81 

This PR adds in the form styling for Step 1 (Store Setup) of the OBW.  p58i-7kf-p2

### Screenshots
<img width="621" alt="screen shot 2018-10-26 at 1 44 10 pm" src="https://user-images.githubusercontent.com/10561050/47586312-71bc8e00-d925-11e8-91de-13fe66591379.png">

### Testing
1.  Visit `/wp-admin/admin.php?page=wc-setup&calypsoify=1`
2.  Check that form styling inside white box matches the new ecommerce plan hifi design
3.  Check that "Add address line 2" displays the input box for "Address line 2"
